src/mcref/RefMigStats.cpp

```diff
@@ -0,0 +1,92 @@
+#include <cstdio>
+#include "RefMigStats.h"
+#include "../patch.h"
+#include "../TauBounds.h"
+#include "../GPhoCS.h"
+#include "../McRefCommon.h"
+
```

Comment:
do you plan to move all McRef source files into a separate directory? Seems like a good idea.